### DISCUSSION TITLE
No places for me

### DISCUSSION THREAD

**Discussion by jaakla** [OvertureMaps/data#29] [Discussion]
I get 404 for s3://overturemaps-us-west-2/release/2023-07-26-alpha.0/theme=places . It is not released yet?

**Comment by sethfitz** [comment]
How are you attempting to access the S3 bucket?

This works for me:

```bash
❯ aws s3 ls --no-sign-request s3://overturemaps-us-west-2/release/2023-07-26-alpha.0/theme=places/
                           PRE type=place/
```
